src/controllers/mainController.ts

```diff
@@ -1951,6 +1987,16 @@ export default class MainController implements vscode.Disposable {
             this._outputContentProvider.onDidCloseTextDocument(doc);
         }
 
+        if (this._actualPlanStatuses.includes(closedDocumentUri)) {
+            const index = this._actualPlanStatuses.indexOf(closedDocumentUri);
+            this._actualPlanStatuses.splice(index, 1); // Remove the URI
```

Comment:
The JS community tends to favor `list.filter(x => x != closedDocumentUri)` since it's more readable